src/questions/expression_control.yaml

```diff
@@ -91,4 +91,576 @@
+  hint: noHint
+  explanation: The required argument should be a conditional statement, hence the data type is boolean.
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: Boolean
+      key: A
+    - content: Integer
+      key: B
+    - content: String
+      key: C
+    - content: Struct
+      key: D
+- uuid: ea3b6f23-b673-4f9a-b928-2bc1ba77ab65
+  type: SingleChoice
+  content: How generally the primary argument withinside the for loop is executed?
+  hint: noHint
+  explanation: The first argument in the for loop is executed only once.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: Infinite
+      key: A
+    - content: One
+      key: B
+    - content: Three
+      key: C
+    - content: It will execute as long as the condition is true
+      key: D
+- uuid: e38441e9-3e06-4910-ba4b-104917d8a641
+  type: SingleChoice
+  content: Is it possible to define single line if/else statement?
+  hint: If/else statments
+  explanation: We can define single if/else statements without curly braces.
+  answerKeys:
+    - C
+  subTopics:
+    - "decision-making"
+  difficultyLevel: Low
+  choices:
+    - content: No it is not possible
+      key: A
+    - content: It is possible only when if/else statements is defined inside function modifiers
+      key: B
+    - content: Yes it is possible
+      key: C
+    - content: None of the above
+      key: D
+
+- uuid: 078cc470-7699-49ac-9122-8b51aa6349ec
+  type: SingleChoice
+  content: How many times the third parameter will execute if the body of for loop is executed n times?
+  hint: noHint
+  explanation: The third parameter will be executed the same number of times as the for loop body.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: n+1 times
+      key: A
+    - content: n times
+      key: B
+    - content: n-1 times
+      key: C
+    - content: one
+      key: D
+
+- uuid: 8144216b-fbef-4520-8f2c-fe9b813e17fc
+  type: SingleChoice
+  content: Why are loops not preferred in solidity?
+  hint: Gas Fees
+  explanation: Loops require a lot of computation, so gas fees can get pretty high.
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: Loops will consume lot of gas for computation
+      key: A
+    - content: Loops will cause bugs in the contract
+      key: B
+    - content: Loops are not preferred as it is complex to use
+      key: C
+    - content: None of These
+      key: D
+
+- uuid: 66b3f13c-5db7-42f6-abf9-ee49dd5e93cc
+  type: SingleChoice
+  content: What is the difference between for loop and while loop?
+  hint: Parameters/Arguments
+  explanation: The for loop requires one parameter and can take two optional parameters. The while loop only requires one parameter, which is a conditional statement.
```

Comment:
The while loop requires only one parameter, and that is the conditional statement.